.changes/1.1.1.md

```diff
@@ -0,0 +1,8 @@
+## 1.1.1 - February 14, 2022
```

Comment:
Now that we have this capability, I like the idea of batching changelog entries in a few different ways:
1. Version-specific changelogs, which also go in GitHub release notes for each prerelease
2. A "combined" changelog for each final release, that batch changes from all prereleases (e.g. v1.1.0b1 + v1.1.0b2 + v1.1.0rc1 = v1.1.0 [final]) — this should go into the final version's release notes
3. A full "combined" changelog, to live continually at `CHANGELOG.md`

To that end, `changie batch` gets us the first, and we should be able to write some dead-simple scripts that get us 2 + 3, by combining the components produced by (1)

Reply:
Hey I found this thread as it was linked to the changie PR above and want to give my two cents. Changie currently has no way of specifying something as pre-release, alpha or similar. My recommendation would be to use the `changie batch --keep` option and move the yaml files into a separate folder before release. Then when the final version is ready, move all the yamls back into unreleased and run batch again. Then 3 can be done by doing `changie merge`. 

If this workflow works for you I would be happy to add it to the Changie roadmap as it is a common workflow when using beta and release candidates. Thanks for considering changes and reach out if you need anything else.

Reply:
Would there be value in incrementally rolling up?  ie v1.1.0b1 rolls into v1.1.0b2 rolls into v1.1.0rc1 rolls into v1.1.0(final) as opposed to not rolling up until the final release?  

changie already has the ability to generate the changelog entry with a `--dry-run` flag when running the `batch` command.  That generates the markdown file that can be merged in but does _not_ delete the associated yaml files.  You would still need to delete the associate entry from the full cahngelog file with this method though so I'm not sure how much it would save over just have a simple script that could accomplish it for us.

Either way, this makes a lot of sense and would really help clean up our `CHANGELOG` as it can be confusing now when there are multiple changlog entries under an rc release and non under the final release (ie [0.21.0](https://github.com/dbt-labs/dbt-core/blob/main/CHANGELOG.md#dbt-0210-october-04-2021)).  Definitely achievable.

Reply:
I'm realizing now rolling up wouldn't be ideal for what gets posted to each individual release.   So scratch that.

I think @miniscruff's suggested approach would work nicely for what we need.  We can either write a script to automate the process he describes or contribute that change back to changie if @miniscruff is open to that.

Reply:
I don't mind adding it if it works for you, just let me know and I can create an issue for it ( or you can start a discussion for ideas )

Reply:
@miniscruff I would really appreciate if you open an issue for this.   It would be a valuable addition for us for sure!